transport/TransportFactory.cpp

```diff
@@ -484,6 +487,44 @@ class TransportFactoryImpl final : public TransportFactory,
         endpoint->stop(this);
     }
 
+    void registerIceListener(Endpoint::IEvents& listener, const std::string& ufrag) override
+    {
+        for (auto endpoints : _sharedEndpoints)
+        {
+            for (auto endpoint : endpoints)
```

Comment:
Less important here but avoid a `shared_ptr` will save some atomic operations on ref count
```suggestion
            for (auto& endpoint : endpoints)
```

Reply:
OK